submissions/Olumide Ologbon/html/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            shopal
+        </title>        
+        <link href = "style.css" rel = "stylesheet">
+    </head>
+    <header class="head">
+        <div class="dropdown">
+            <button class="dropbtn">More</button>
+            <div class="dropdown-ctn">
+            <a href="index.html.html">Home</a>
+            <a href="#">About</a>
+            <a href="#">Clients</a>
+            <a href="#">Contact Us</a>
+            </div>
+        </div>   
+        <div class="dropdown">
+                <button class="dropbtn">Account</button>
+            <div class="dropdown-ctn">
+                    <a href="signup.html">Sign up </a>
+                    <a href="#">Login</a>
+                    <a href="#">Track Orders</a>
+                    <a href="#">Wish List</a>
+            </div>
+        </div>        
+        <h1>
+            Shopal Online
+        </h1>
+    </header>
+    <body>
+        <div>   
+        <img src="C:\Users\OLOGBON\Desktop\work\Dufuna\dufuna-html/plate.jpg" alt="promo!!!" class="img1">
```

Comment:
You need to use a relative path for your images, this would only work on your own computer

Reply:
Same applies to all the other images you have in this PR

Reply:
> You need to use a relative path for your images, this would only work on your own computer

i added all the images to a folder in the html folder. how do use a relative path for the image?

Reply:
i understand what to do now. i will work on it. thanks

Reply:
i have made updates